client/src/context/AppProvider.js

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useState } from 'react';
+
+export const AppContext = createContext();
+
+const AppProvider = ({ children }) => {
```

Comment:
Are you sure it is not worth to replicate any state manager with actions and atomic changes using `useReduce`?
Just for now it looks pretty clear but the more data entities you would add here, the worse this module will looks.

Reply:
@Xoxol I decided not to use any state manager because it's really small app with few pages.